src/main/java/com/foundation/controller/Controller.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * @(#)Controller.java Copyright (c) 2018 Jalasoft.
+ *  2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ *  All rights reserved.
+ *  <p>
+ *  This software is the confidential and proprietary information of
+ *  Jalasoft, ("Confidential Information").  You shall not
+ *  disclose such Confidential Information and shall use it only in
+ *  accordance with the terms of the license agreement you entered into
+ *  with Jalasoft.
+ */
+
+package com.foundation.controller;
+
+import java.text.ParseException;
+
+/**
+ * Class to integrate View and Search classes
+ *
+ * @author Marco Velasquez
+ * @version 1.0.
+ */
+public class Controller {
+
+    Validator validate;
+    SearchCriteria criteria;
+    // View view;
+    // Search search;
+    private String path;
+    private String fileName;
+    private String fileType;
+    private String visibility;
+    private String fileSize;
+    private String dateModified;
```

Comment:
Do you really need all these attributes in the Controller class? I think no, the controller shouldn't have those attributes, you can work only with local variables in the method level and use the setter for the SearchCriteria class